web/theme/templates/base.html

```diff
@@ -11,7 +11,7 @@
       {#{% assets "sass" %}#}
       <link rel="stylesheet" href="{{ url_for('static', filename='css/app.css') }}">
       <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-      <link rel="stylesheet" href="//okfnlabs.org/ok-panel/assets/css/frontend.css"/>
+      <link rel="stylesheet" href="//cdn.rawgit.com/okfn/ok-panel/2.1.0/assets/css/frontend.css"/>
```

Comment:
why are you using rawgit? if we are linking to assets, we should do it from the new repo `a`